phpstan.el

```diff
@@ -181,10 +181,12 @@ NIL
           (expand-file-name (cdr phpstan-config-file) (php-project-get-root-dir))
         phpstan-config-file)
     (let ((working-directory (phpstan-get-working-dir)))
-      (cl-loop for name in '("phpstan.neon" "phpstan.neon.dist")
-               for dir  = (locate-dominating-file working-directory name)
-               if dir
-               return (expand-file-name name dir)))))
+      (if working-directory
+          nil
```

Comment:
This line is mistake.  Fixed in https://github.com/emacs-php/phpstan.el/commit/e32ef8ef696fbc5165a9382c2ec218d9f03209a0